src/storage/ResourceStore.ts

```diff
@@ -4,6 +4,55 @@ import type { RepresentationPreferences } from '../http/representation/Represent
 import type { ResourceIdentifier } from '../http/representation/ResourceIdentifier';
 import type { Conditions } from './Conditions';
 
+/**
```

Comment:
Naming in this module should be inline with the rest (creation functions should start `create…`)

Reply:
This is now corrected in (https://github.com/solid/community-server/pull/1075/commits/461a9361621353f3af3b65d09dddb2d633e96efa)